ci/nightly/pipeline.template.yml

```diff
@@ -85,7 +85,7 @@ steps:
                - common-ancestor
     - id: scalability-benchmark
       label: "Scalability benchmark against merge base or 'latest'"
-      timeout_in_minutes: 180
+      timeout_in_minutes: 300
```

Comment:
That seems long! Why does this additional test take so long?

Reply:
Though, let me await the current nightly build to see the latest duration.

Reply:
It took 2.5 hours, that's 0.5 hours more compare to a build on `main`. 180 minutes should be sufficient...

Reply:
Reverted this change, a further run in nightly succeeded timely.